README.md

```diff
@@ -34,7 +34,9 @@ A very basic Sculpin based blog supporting the following features:
  * A blog tags page at `/blog/tags`.
  * A blog tag index at `/blog/tags/$tag`. Similar to the blog archive
    except broken down by each tag.
-
+ * Authors bios can be found in /views/partials/first-last
+ * Author bio photos should be saved in /images/bio-pics
+ * Adding new author tags for relevent posts is in /views/partials.  Within will be instructions on how to add new authors.
```

Comment:
Let's have a heading that reads something like "Adding a new author" and then lists step by step the files that need to be edited because some are missing here:
1. add bio to views/partials/xyz-asdlfkjf
2. save author image in xyz
3. Create a blog post with the author tag in the tags section like "xyz-zyx"
4. update array list here: source/_views/partials/tags.html
5. add conditional ehre 
source/authors/index.html (though this should probably be a loop iterating over everyone in an authors array.)